docs/my-website/docs/proxy/alerting.md

```diff
@@ -33,13 +34,11 @@ general_settings:
     alerting: ["slack"]
     alerting_threshold: 300 # sends alerts if requests hang for 5min+ and responses take 5min+ 
 
+environment_variables:
+    SLACK_WEBHOOK_URL: "https://hooks.slack.com/services/<>/<>/<>"
+    SLACK_DAILY_REPORT_FREQUENCY: "86400"  # 24 hours; Optional: defaults to 12 hours
```

Comment:
please can you add more detail on what the daily reports are?  A user might be unsure about what this param means / controls

Reply:
Sure! How about 464545114?

Reply:
awesome ! thanks

Reply:
![Screenshot 2024-05-13 at 9 06 56 PM](https://github.com/BerriAI/litellm/assets/305268/42641946-df3e-4e59-a6e1-cce25a5c7d49)